dbt/adapters/bigquery/impl.py

```diff
@@ -261,6 +261,29 @@ def list_relations_without_caching(
             logger.debug("list_relations_without_caching error: {}".format(str(exc)))
             return []
 
+    @available
+    def standardize_grants_dict(self, grants_table: agate.Table) -> dict:
```

Comment:
If this is actually the exact same as Postgres' version, should we just make this the `base` implementation?

Question also for @McKnight-42 @gshank

Reply:
➕